src/preqx/prim_advection_mod.F90

```diff
@@ -4,7 +4,7 @@
 
 module prim_advection_mod
 #ifndef USE_KOKKOS_KERNELS
-  use prim_advection_mod_base, only: Prim_Advec_Tracers_remap_ALE, prim_advec_tracers_fvm, vertical_remap_interface, Prim_Advec_Tracers_remap, Prim_Advec_Tracers_remap_rk2
```

Comment:
Andrew  will address this better, but i think _ALE routine is in leap tracers...

Reply:
It's OK to get ride of these. A lot has changed in the SL code between XX and E3SM repos. I'll eventually add back what is needed.